### PR TITLE
add enzyme list to morpheus tool

### PR DESCRIPTION
**Pull request by mthang**
FOR CONTRIBUTOR:

[x]I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
[x]License permits unrestricted use (educational + commercial)
[x]This PR adds a new tool or tool collection
[x]This PR updates an existing tool or tool collection
[]This PR does something else (explain below)

Background:
The morpheus wrapper does not have the parameter (p) which allows user to select enzyme to be used in the morpheus search process. The option p is listed in https://cwenger.github.io/Morpheus/args.html . 

In addition to include the p option in the wrapper. I have tidied up the morpheus wrapper by creating a macro file for tool version , tokens, values and enzyme list for the paramters used in the wrapper.

### CODE REVIEW

File: tools/morpheus/morpheus.xml [hunk 1]
```diff
@@ -1,49 +1,11 @@
-<tool id="morpheus" name="Morpheus" version="2.255.0">
+<tool id="morpheusid" name="Morpheus" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="@PROFILE@">
```
Comment (bgruening) [inline tools/morpheus/morpheus.xml:1]: please don't change the tool ID, otherwise we loose the tool lineage

File: tools/morpheus/macros.xml [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<macros>
+        <token name="@TOOL_VERSION@">2.255.0</token>
```
Comment (bgruening) [inline tools/morpheus/macros.xml:2]: the version needs to be bumped

**Review by bgruening — COMMENTED** [review]
Thanks @mthang, 2 comments inline.

File: tools/morpheus/morpheus.xml (hunk 1 above)
Comment (mthang) [reply tools/morpheus/morpheus.xml:1]: My bad. I forgot to change it back after testing.

File: tools/morpheus/macros.xml (hunk 2 above)
Comment (mthang) [reply tools/morpheus/macros.xml:2]: agreed! The latest version is 288. do you want me to bump it up to 288?

Comment (bgruening) [reply tools/morpheus/macros.xml:2]: Thats a good idea.

**Comment by bgruening** [comment]
Please also see the failing lint: 

> .. WARNING: Missing shed metadata field [homepage_url] for repository

File: tools/morpheus/macros.xml (hunk 2 above)
Comment (mthang) [reply tools/morpheus/macros.xml]: @bgruening I bumped the version up to 288 from 255 and updated the output of the test section. Can you check again.

File: tools/morpheus/morpheus.xml [hunk 3]
```diff
@@ -150,7 +112,10 @@
         #if str($vm) != 'None':
             #set $vmods = str($vm).replace(',',';')
             -vm="$vmods"
-        #end if
+	#end if
+	#if str($p) != 'None':
```
Comment (bgruening) [inline tools/morpheus/morpheus.xml:116]: It can never be None, because the select param is not optional.

**Review by bgruening — APPROVED** [review]
Looks good to me! Thanks @mthang

**Pull request merged**